lib/benchee/formatters/console.ex

```diff
@@ -145,11 +155,65 @@ defmodule Benchee.Formatters.Console do
     units = Conversion.units(sorted_scenarios, scaling_strategy)
     label_width = label_width(sorted_scenarios)
 
-    [column_descriptors(label_width) |
+    output = [column_descriptors(label_width) |
       scenario_reports(sorted_scenarios, units, label_width)
       ++ comparison_report(sorted_scenarios, units, label_width, config)]
+
+    if config.extended_options do
```

Comment:
Instead of the if/else I'd rather rely on a pattern match like `comparison_report` if feasible. i.e. just return an empty array and do `++ extended_statistics_report(args)`